cache.go

```diff
@@ -40,3 +40,17 @@ func Cache(table string) *CacheTable {
 
 	return t
 }
+
+// RemoveCache flushes and removes the table if it exists.
+func RemoveCache(table string) {
+	mutex.RLock()
+	defer mutex.RUnlock()
+
+	// since we remove it, its cache is unnecessary ao we flush it.
+	cacheTable := cache[table]
+	if cacheTable != nil {
+		cacheTable.Flush()
```

Comment:
Is flushing really necessary here? If we drop the last reference to the table, the GC should clean up those values for us.

Reply:
Yes. It should be unnecessary. I remember it is trick for java that can release those objects as early as possible, but I think it should be unnecessary for go.